www/js/index.js

```diff
@@ -110,7 +112,67 @@ openSonect: function() {
     )
 },
 
-paySonect: function() {
+openSonectWithPaymentReference: function() {
```

Comment:
`openSonectWithPaymentReference` and `paySonectiOS` do the exact same thing, pass the payment ref to the SDK. Can you somehow store what the credentials, value and currency were between Show / Hide the SDK (You could probably store it in the plugin code)? I use `presentTransaction` to show the SDK again with payment reference? That would probably completely unify the JS interface then